hack/apt/build_package_repo.sh

```diff
@@ -83,6 +100,7 @@ SHA1: $(sha1sum ${PKG} | cut -f1 -d' ')
 MD5sum: $(md5sum ${PKG} | cut -f1 -d' ')
 SHA512: $(sha512sum ${PKG} | cut -f1 -d' ')
 Description: The core Tanzu CLI
+
```

Comment:
nit: don't think it matters anymore now that we have the cleanser, but would the right thing be to add a blank line here?

Reply:
Yes.  But it is added, right?  Empty "+" line.

Reply:
duh, your are right.